render-test/metadata.hpp

```diff
@@ -82,13 +88,33 @@ struct NetworkProbe {
     size_t transferred;
 };
 
+struct GfxProbe {
+    struct Memory {
+        int allocated = 0;
```

Comment:
size_t

Reply:
Memory tracking between probeGFXStart and probeGFXEnd is relative, ie. these numbers could be negative in a tracking segment where more memory was released than allocated. I might need to rethink the logic :)